python/amici/ode_export.py

```diff
@@ -1494,18 +1494,19 @@ def _totalDerivative(self, name, eq, chainvar, var,
             else:
                 variables[var]['sym'] = self.eq(varname)
 
+        self._eqs[name] = \
+            self._eqs[name] = copy.deepcopy(variables['dydz']['sym'])
```

Comment:
@dweindl note that a you need to deepcopy here, did not spot this when I reviewed your code.

Reply:
Thx. Will merge fix with #500.